komodor/integrations.go

```diff
@@ -0,0 +1,64 @@
+package komodor
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+const IntegrationsUrl string = DefaultEndpoint + "/integrations/kubernetes"
+
+type Kubernetes struct {
+	Id          string `json:"apiKey"`
+	ClusterName string `json:"clusterName"`
```

Comment:
I agree that it is used in the Get and Create, but it is only being set in those functions,
the only usage of the `Kubernetes.ClusterName` is in `resource_komodor_kubernetes.go:42` for log, and in this func, you already have a `clusterName` variable 9 lines before.

Additionally, the clusterName value is provided by the client and is not returned in the API.

I believe it is safe to remove it (unless I'm missing something...)

Reply:
yeah misunderstood what you were saying originally. I've removed it from the struct.